src/main/java/org/team199/robot2021/subsystems/Intake.java

```diff
@@ -77,5 +81,17 @@ public void doTheFlop() {
     public boolean isDeployed() {
         return deployed;
     }
+    public boolean isLimelightSearching() {
+        return isLimelightSearching;
+    }
+    public void changeServoStatus(boolean status) {
+        isLimelightSearching = status;
+        if (isLimelightSearching() == false) {
+            limelightServo.setAngle(kLimeBottomAngle); // TODO: set correct angle
+        } else {
+            limelightServo.setAngle(kLimeTopAngle); // TODO: set correct angle
+        }
+    }
```

Comment:
This will actually set the limelight to the opposite state of `isLimelightSearching`. If `true` is passed into the function (indicating that the Limelight should be searching), `isLimelightSearching` will become `true`, so `isLimelightSearching == false` will be `false`. Thus, `limelightServo.setAngle(kLimeTopAngle);` will be called instead of `limelightServo.setAngle(kLimeBottomAngle);`